### PR TITLE
Front end

### PR DESCRIPTION
**Pull request by zceaton**

### CODE REVIEW

File: Front-End/Angular-JS/README.md [hunk 1]
```diff
@@ -0,0 +1,295 @@
+# `angular-seed` — the seed for AngularJS apps
```
Comment (abarganier) [inline Front-End/Angular-JS/README.md]: I think we should ditch the majority of this README, what do you think? If there are parts of this that are relevant to building the front end of the project (i.e. info on the `Angular-JS/app/index-async.html` file) then we can leave those in, but I think the majority of this content isn't necessarily relevant to our project.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +0,0 @@
-.idea/
```
Comment (abarganier) [inline .gitignore]: Let's add this original .gitignore back into the root and then merge the angular-specific one into the root file.

File: Front-End/Angular-JS/README.md (hunk 1 above)
Comment (zceaton) [reply Front-End/Angular-JS/README.md]: Yeah, I agree.  I didn't mean to commit the README from the other project, I'll take a look to see what we should keep and what should be removed.

**Comment by zceaton** [comment]
Removed the Angular-JS directory and merged the .gitignore files in the directory, I'll commit all the changes when I get the chance to update the README.

**Pull request merged**